prefetch.bs

```diff
@@ -90,8 +90,8 @@ The user agent may remove elements from the [=prefetch buffer=] even if they are
     1. [=list/Append=] a [=prefetch record=] with [=prefetch record/URL=] |url|, [=prefetch record/referrer policy=] |referrerPolicy|, [=prefetch record/sandbox flags present=] true if |sandboxFlags| is not empty and false otherwise, [=prefetch record/redirect chain=] |redirectChain| and [=prefetch record/expiry time=] |expiryTime| to |document|'s [=prefetch buffer=].
 </div>
 
-<div algorithm="find a matching prefetch response">
-    To <dfn export>find a matching prefetch response</dfn> given a {{Document}} |document|, [=URL=] |url|, [=referrer policy=] |referrerPolicy| and [=sandboxing flag set=] |sandboxFlags|, perform the following steps.
+<div algorithm="find a matching prefetch record">
```

Comment:
FYI the `algorithm=""` shouldn't be necessary since you only have one `<dfn>` inside the `<div>`. It's for the patch cases where we have zero `<dfn>`s where it becomes important.

Reply:
Removing `algorithm="..."` it breaks cross-references for variables within the algorithm.

Reply:
Sorry, I meant that you can just do `<div algorithm>`, instead of `<div algorithm="stuff">`